src/test/java/jenkins/jmh/benchmarks/JmhStateBenchmark.java

```diff
@@ -0,0 +1,18 @@
+package jenkins.jmh.benchmarks;
+
+import jenkins.jmh.JmhBenchmark;
+import jenkins.jmh.JmhBenchmarkState;
+import org.openjdk.jmh.annotations.Benchmark;
+
+import java.util.Objects;
+
+@JmhBenchmark
+public class JmhStateBenchmark {
```

Comment:
Do you want to have it just a for a sample? Fine as a smoke test

Reply:
Yes, this is just a sample.